dataproxy/vercel-cli/prisma/schema.prisma

```diff
@@ -0,0 +1,31 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATAPROXY_VERCEL_CLI_DB_URL")
```

Comment:
Where is this database hosted?

Reply:
It's the same db as all the other deployments, so provisioned by us and then put behind the data proxy

Reply:
Ah ok, so this is a data proxy specific one?

That also explains why we only read and not write I guess (other comment).